public/sass/elements/_forms.scss

```diff
@@ -223,3 +224,13 @@ textarea.form-control {
     width: 12.5%;
   }
 }
+
+// 7. Browser accessibility fixes
+// ==========================================================================
+
+option:active,
```

Comment:
Are you sure `:active` is needed? My gut feeling said `:checked` is weird but from testing all 3 of them just now, it seems like `:checked` is needed for the activate state when the select box is open and `:focus::-ms-value` is needed for the active state when the select box is closed (but still focussed). Do you know for what case `:active` is needed?